backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service-admin-use-cases.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-self-service-admin-use-cases"]
+= {oadp-short} Self-Service admin use cases
+include::_attributes/common-attributes.adoc[]
+:context: oadp-self-service-admin-use-cases
+
+toc::[]
+
+As a cluster administrator, following are some backup and restore scenarios where you enable {oadp-short} Self-Service and configure the `DataProtectionApplication` according to the type of the backup:
+
+* Filesystem backup and restore by using Kopia.
+* Data Mover backup and restore.
+* Container Storage Interface (CSI) backup and restore.
```

Comment:
So FWIW..  The only difference between b/r w/ kopia ( don't use that term please FSB)  and DM here is what the DEFAULT is set to in the DPA.    I don't think you need seperate sections to delineate the two types of backups.  Same thing w/ CSI..

In any of these cases if a user constructs the a backup to for DM and the DPA has default to use FSB, the backup will use DM.   The default only comes into play WHEN the type of backup is NOT specificied in the backup :)   @shubham-pampattiwar WDYT?